examples/hang_fuzz_test.cc

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ *Licensed under the Apache License, Version 2.0 (the "License");
+ *you may not use this file except in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing, software
+ *distributed under the License is distributed on an "AS IS" BASIS,
+ *WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *See the License for the specific language governing permissions and
+ *limitations under the License.
+ */
+
+// A fuzz target that hangs.
+
+#include <stdint.h>
+#include <stddef.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  while (true)
+  {
```

Comment:
Place the opening brace on the same line: https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace